### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Dongmo12**

### CODE REVIEW

File: spec/game_logic_spec.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
+require './lib/game_logic'
```
Comment (Ispirett) [inline spec/game_logic_spec.rb]: You can use `requre_relative` instead require  for files in your directory

File: spec/player_spec.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+require "./lib/player"
+
+RSpec.describe Player do
+  it 'Tests the player name' do
+    player = Player.new('player', 'T')
+    expect(player.name).to eql('player')
+  end
+
+  it 'Tests the player symbols' do
+    player = Player.new('player', 'T')
```
Comment (Ispirett) [inline spec/player_spec.rb]: You can use repeating this you can write **let(player) {Player.new('player', 'T'}**

**Review by Ispirett — CHANGES_REQUESTED** [review]
Great work here @Dongmo12 @ibrolive   with the testing.
image. 

There are some tests that are missing,

- [x] You should test for winners player one and two
- [x] You should test for players drawing 
- [x] You should test for other futures of the game,  example player uses a number that is not in range.

**When finished submitting for re-review**
## Git reminder.

Friendly reminder: You should use Git Flow in this project. However, changing it now would take too much time.
Please make sure you use Git Flow in the next project. You will find instructions in the Patwright lesson description.

File: spec/game_logic_spec.rb (hunk 1 above)
Comment (Dongmo12) [reply spec/game_logic_spec.rb]: Thank you for this!!

It has been handled.

File: spec/player_spec.rb (hunk 2 above)
Comment (Dongmo12) [reply spec/player_spec.rb]: Thanks!!

**Review by Torres-ssf — APPROVED** [review]
### Hi there team :hand: 

Nice job implementing the suggested changes. The tests are working nice :+1: 

**[FRIENDLY WARNING]**
As it said on the [pathwrite](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/), we need to use [Git Flow](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59882084/) instead of Github flow. And as Emily announced in the [full-time-students](https://microverse-students.slack.com/archives/GA812K2TC/p1569840656127800) today, Github/Git flow became a [hard requirement](https://microverse.zendesk.com/hc/en-us/articles/360036559533-9-30-2019-Github-or-Git-Flow-becomes-a-hard-requirement). The purpose of this is that almost every development team in the world uses this approach, so this will definitely benefit your chances of landing your first job, and will also make your repo look more professional. **Since this is not the first review in this PR, I'm approving it :+1:**. But for next time we need to use Git flow or we TSEs are not allowed to review.

Good luck with the next project. Happy coding :computer:

**Pull request merged**